altiProfil/lib/AltiServicesFromIGN.php

```diff
@@ -40,18 +40,18 @@ public function getAlti($lon, $lat)
 
         $urlAltiIGN = $this->config->getIgnServiceUrl($APIRestElev, $data);
 
-        list($data, $mime, $code) = \lizmapProxy::getRemoteData($urlAltiIGN);
+        list($data, $mime, $code) = \Lizmap\Request\Proxy::getRemoteData($urlAltiIGN);
         $code = 200;
         if ($code == 200) {
             //DATA SHOULD BE LIKE '{"elevations":[{"x":55.38025625,"y":-21.14050849,"z":2154.75,"acc":2.5}]}'
             /* FOR TESTING
             $data = '{"elevations":[{"x":55.38025625,"y":-21.14050849,"z":2154.75,"acc":2.5}]}' ;
             */
-            return $data;
+            return json_decode($data);
         }else{
             $errorMsg = "AltiProfil IGN wrong request";
             \jLog::log($errorMsg);
-            return '{"error msg": "'.$data.'" }';
```

Comment:
Side note, but it would be nice to edit the docstring at the same time, to help IDE with functions, code quality, QA tools such as PHP Stan (later, not yet added I know) ;-)